crates/cli-support/src/js/binding.rs

```diff
@@ -76,6 +78,14 @@ pub struct JsFunction {
     pub log_error: bool,
 }
 
+/// A references to an (likely) exported symbol used in TS type expression.
+///
+/// Right now, only string enum require this type of anaylsis.
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+pub enum TSReference {
```

Comment:
```suggestion
pub enum TsReference {
```
See the [API Guidelines](https://rust-lang.github.io/api-guidelines/naming.html#casing-conforms-to-rfc-430-c-case).